docs/modules/con_thorntail-yaml-project-stages.adoc

```diff
@@ -0,0 +1,80 @@
+
+[id='the-thorntail-yaml-project-stages_{context}']
+= Configuration overlays using stage properties
+
+In some cases it may be desirable to extract some of the configuration properties and switch them depending on the target environment. Project stages are a way to move environment specific configuration properties into an external file.
+
+.Stage configurations in project-defaults.yml
+====
+This example shows three stages, `default`, `development` and `production` being configured in the default project-defaults.yml.
```

Comment:
Do we still want to encourage this? I'm not even sure it's tested :-)

Reply:
I guess having a single file overlaying the properties may be reasonable for a start, for the users to get a quick try at this approach before starting creating multiple files...